mllib/src/test/scala/org/apache/spark/ml/classification/ProbabilisticClassifierSuite.scala

```diff
@@ -91,4 +94,60 @@ object ProbabilisticClassifierSuite {
     "thresholds" -> Array(0.4, 0.6)
   )
 
+  /**
+   * Add test for prediction using the model with all combinations of
```

Comment:
Tiny nit: This could be reworded from the JIRA description.

How about:

Helper for testing that a ProbabilisticClassificationModel computes the same predictions across all combinations of output columns (rawPrediction/probability/prediction) turned on/off. Makes sure the output column values match by comparing vs. the case with all 3 output columns turned on.